.github/workflows/on_pull_request.yaml

```
@@ -0,0 +1,18 @@
+name: On Pull Request
+
+on:
+  pull_request:
+
+jobs:
+
+  on-pull-request:
+    name: Get rocks modified and build-scan-test them
+    uses: canonical/charmed-kubeflow-workflows/.github/workflows/get-rocks-modified-and-build-scan-test-publish.yaml@main
+    permissions:
+      pull-requests: read
+    secrets: inherit
+    with:
+      microk8s-channel: 1.32-strict/stable
+      juju-channel: 3.6/stable
+      python-version: "3.9"
```

This will probably not be used (as in the other repos), but let's stick with the version we use in other rocks (probably `3.8` or `3.10`)
```suggestion
      python-version: "3.9"
```